src/hw/actions/transaction.js

```diff
@@ -129,6 +134,18 @@ export const createAction = (
 
       const bridge = getAccountBridge(mainAccount);
 
+      log(
+        "transaction-summary",
+        `→ FROM ${formatAccount(mainAccount, "basic")}`
+      );
+      log(
+        "transaction-summary",
+        `✔️ transaction ${formatTransaction(transaction, mainAccount)}`
+      );
+
+      // log("transaction-summary", `STATUS ${formatTransactionStatus(transaction, status, mainAccount)}`);
+      // status, how to get it ?
```

Comment:
it's a good point. maybe it means we are not doing this at the correct place. but now where would that be🤔

on LLD/LLM side we indeed will use `bridge.broadcast` which is implementation dependant so we don't want to do it N time for each family.

However, i wonder if we could always force us to use a higher level "broadcast". which we already do here: 
https://github.com/LedgerHQ/ledger-live-desktop/blob/766fdf662b78d1d9824367687a5ffe725feef730/src/renderer/hooks/useBroadcast.js#L21

indeed we also need to handle DISABLE_TRANSACTION_BROADCAST which is a generic thing too.

it sounds like we could generalize it back into live common and have the logs in this place. (also the signed op log 🤔  )

so to me it sounds like this broadcast is to be in live-common.

I just know that platform started doing a tiny differ version of it, so maybe we also need to have this in scope (i think we need the non hook version in live-common at least).